detections/endpoint/windows_suspicious_process_file_path.yml

```diff
@@ -20,7 +20,7 @@ data_source:
 search: '| tstats `security_content_summariesonly` count values(Processes.process_name)
   as process_name values(Processes.process) as process min(_time) as firstTime max(_time)
   as lastTime from datamodel=Endpoint.Processes 
-  where Processes.process_path IN("*\\windows\\fonts\\*", "*\\users\\public\\*", "*\\windows\\debug\\*", "*\\Users\\Administrator\\Music\\*", "*Recycle.bin*", "*\\Windows\\Media\\*","\\Windows\\repair\\*", "*\\PerfLogs\\*", "*:\\Windows\\Prefetch\\*", "*:\\Windows\\Cursors\\*", "*:\\Windows\\INF\\*") AND NOT(Processes.process_path IN ("*\\temp\\*"))
+  where Processes.process_path IN("*\\windows\\fonts\\*", "*\\users\\public\\*", "*\\windows\\debug\\*", "*\\Users\\Administrator\\Music\\*", "*Recycle.bin*", "*\\Windows\\Media\\*","\\Windows\\repair\\*", "*\\PerfLogs\\*", "*:\\Windows\\Prefetch\\*", "*:\\Windows\\Cursors\\*", "*:\\Windows\\INF\\*", "*\\programdata\\*") AND NOT(Processes.process_path IN ("*\\temp\\*"))
```

Comment:
`ProgramData` is a legitimate path with many legitimate binaries executing from there.

```suggestion
  where Processes.process_path IN("*\\windows\\fonts\\*", "*\\users\\public\\*", "*\\windows\\debug\\*", "*\\Users\\Administrator\\Music\\*", "*Recycle.bin*", "*\\Windows\\Media\\*","\\Windows\\repair\\*", "*\\PerfLogs\\*", "*:\\Windows\\Prefetch\\*", "*:\\Windows\\Cursors\\*", "*:\\Windows\\INF\\*") AND NOT(Processes.process_path IN ("*\\temp\\*"))
```

Reply:
This is not resolved yet.

Reply:
That indeed is correct , ProgramData is legitimate dir! committing this suggestion!

Reply:
@nasbench That said, upon futhur reading it does looks like ProgramData is oddly used by malwares (example bad browser extensions) to drop files into but I think it does make sense to leave it in here ? 🤷 

example: https://forums.malwarebytes.com/topic/307988-malicious-exe-files-constantly-generating-inside-my-programdata-folder/

Reply:
It make sense to monitor it but not to leave it in that rule. That rule is a TTP and in real life more often than not, all alerts generated from it would FPs.

Reply:
Random example from one of my labs of things that execute from ProgramData (including windows defender)
![image](https://github.com/user-attachments/assets/05783d7f-ac83-46d5-9e59-1a9c20dfd72e)

Reply:
hmmmn makes sense! also better to have this catch specific things in others and remove it from this rule in lieu of catching "other" behaviour with high fidelity   !

We need to whitelist you @nasbench ! 🥇